getting-started/about-telerik-reporting.md

```diff
@@ -11,9 +11,7 @@ position: 0
 
 # Basic Concepts in Telerik Reporting
 
-Telerik Reporting is an [embedded tool](https://www.telerik.com/products/reporting/embedded-reporting.aspx) for creating and displaying reports in a visually structured and logically grouped way.
```

Comment:
I think that the SEO team wanted us to have a link to the page https://www.telerik.com/products/reporting/embedded-reporting.aspx